content/maxwell3_fdem/sphere_freespace.rst

```diff
@@ -0,0 +1,262 @@
+
+.. math::
+	\vec F (\omega) = \begin{cases}
+	F_b e^{i \omega t} \hat z \; \; \textrm{  at  } \; \; r>R \\
+	\\
+	F_s e^{i \omega t} \hat z \; \; \textrm{  at  } \; \; r<R 
+	\end{cases}
+	:label: SolnsInsideOutside
+
+
+For our problem, boundary conditions on the sphere require that tanjential components of the magnetic field and normal components of the flux density must be continuous.
+According to Wait (1951), these conditions are satisfied by the following:
+
+.. math::
+	\textrm{At }r=R: \; \begin{cases}
+	\dfrac{1}{r} \dfrac{\partial F_b}{\partial r} - \gamma_b^2 F_b = \dfrac{1}{r} \dfrac{\partial F_s}{\partial r} - \gamma_s^2 F_s \\
+	  \\
+	\mu_b \Bigg ( \dfrac{\partial^2 F_b}{\partial r^2} - \gamma_b^2 F_b \Bigg ) = \mu_s \Bigg ( \dfrac{\partial^2 F_s}{\partial r^2} - \gamma_s^2 F_s \Bigg )
+	\end{cases}
+	:label: BoundaryConditions
+
+To solve the boundary value problem, Wait (1951) expressed the Schelkunoff potentials, both inside and outside of the sphere, as a sum of spherical harmonic modes with coefficients :math:`a_n` and :math:`b_n`, respectively.
+For the boundary conditions to be satisfied however, he found that coefficients :math:`a_n=b_n=0 \; \forall \; n>0`.
+As a result, the solution to the Schelkunoff potentials inside and outside the sphere are defined by:
+
+.. math::
+	F_b = - \frac{H_0 }{\sigma_b + i \omega \varepsilon_b} + i \omega \mu_b  \frac{e^{-\gamma_b r}}{r}a_0 H_0
+	:label: Foutside
+
+and
+
+.. math::
+	F_s = i \omega \mu_s \frac{sinh \big ( \gamma_s r \big )}{r} b_0 H_0
+	:label: Finside
+
+To determine the solution outside of the sphere, Eqs. :eq:`Foutside` and :eq:`Finside` may be substituted into Eq. :eq:`BoundaryConditions`.
+Through meticulous algebra, coefficient :math:`a_0` can be expressed as:
+
+.. math::
+	a_0 \! =\! \frac{R^3}{2 e^{-\alpha_b}} \!\Bigg [ \! \frac{2\mu_s \big [ tanh(\alpha_s) - \alpha_s  \big ] + \mu_b \big [\alpha_s^2 \, tanh(\alpha_s) - \alpha_s + tanh(\alpha_s) \big ] }{\mu_s \big ( \alpha_b^2 +\alpha_b + 1 \big ) \big [ tanh(\alpha_s) - \alpha_s \big ] - \mu_b \big ( \alpha_b + 1 \big ) \big [ \alpha_s^2 \, tanh(\alpha_s) - \alpha_s + tanh(\alpha_s) \big ] } \! \Bigg ]
+	:label: a0
+	
+where
+
+.. math::
+	\alpha_b = \gamma_b R = \Big [ i \omega \mu_b \sigma_b - \omega^2 \mu_b \varepsilon_b \Big ]^{1/2} R
+	:label: alpha_b
+	
+and
+
+.. math::
+	\alpha_s = \gamma_s R = \Big [ i \omega \mu_s \sigma_s - \omega^2 \mu_s \varepsilon_s \Big ]^{1/2} R
+	:label: alpha_s
+
+The total magnetic field outside the sphere, in response to an inducing field of the form :math:`\vec H_0 e^{i\omega t}`, may be obtained by substituting Eqs. :eq:`Foutside` and :eq:`a0` into Eq. :eq:`SchelkunoffH`.
+Note that our derivation of :math:`a_0` did not require us to include the frequency-dependent term :math:`e^{i\omega t}` of the primary field.
+Therefore, we may generalize our solution for any inducing field of the form :math:`\vec H_0 (i\omega )`.
```

Comment:
So what does the solution looks like then and how it is different from the dipole response presented later? (is the dipole response can be understood as the secondary field, compared to the primary and total fields?)